package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "solc",
+  "version": "0.1.3-1",
```

Comment:
Not really, it's only really used at publish time.

It may be work just having the version represent the npm module itself not solidity. not sure if this would be more or less confusing.